### PR TITLE
Update gallery.html to 4 grids and add entry for GBR. Fixed page resp…

### PR DESCRIPTION
**Pull request by ucpasas**
…onse issue.

### CODE REVIEW

File: buildscripts/pages/gallery.html [hunk 1]
```diff
@@ -44,15 +44,15 @@ <h1 class="d-none d-lg-block">udStream Showcase</h1>
...
 				<div class="card">
 					<a v-bind:href="project.project" target="framebox">
 						<div class="row no-gutters">
-							<div class="col-sm-12 col-md-6 col-xl-4" style="margin: auto; text-align:center;">
+							<div class="col-sm-12 col-md-8 col-xl-4" style="margin: auto; text-align:center;">
 								<img v-bind:src="project.image" class="card-img" v-bind:alt="project.name" style="width: 128px; height: 128px;">
 							</div>
-							<div class="col-md-6 col-xl-8 d-none d-md-block">
-								<div class="card-body p-1 m-1">
+							<div class="col-md-3 col-xl-6 d-none d-md-block">
```
Comment (OsiJr) [inline buildscripts/pages/gallery.html:54]: `col-md-4` ? With the `8` above to get to 12. `col-xl-6` is either wrong here or above as well (to get to 12)

File: buildscripts/pages/gallery.html [hunk 2]
```diff
@@ -44,15 +44,15 @@ <h1 class="d-none d-lg-block">udStream Showcase</h1>
...
+								<div class="card-body p-1 m-0.1">
```
Comment (OsiJr) [inline buildscripts/pages/gallery.html:55]: `m-0.1` isn't specified in the list https://getbootstrap.com/docs/4.0/utilities/spacing/

File: buildscripts/pages/gallery.html [hunk 3]
```diff
@@ -84,10 +84,16 @@ <h1 class="d-none d-lg-block">udStream Showcase</h1>
 				description: "This shows how you can display underground data."
 			},
 			{
-				name: "South-East Queensland",
+				name: "SE Queensland",
```
Comment (OsiJr) [inline buildscripts/pages/gallery.html:87]: `SE` means nothing to anyone, you're better off removing it entirely and just having "Queensland"? "SEQ" means something to Queenslanders but not the rest of the world.

**Review by OsiJr — CHANGES_REQUESTED** [review]
See requested changes as above

File: buildscripts/pages/gallery.html (hunk 1 above)
Comment (ucpasas) [reply buildscripts/pages/gallery.html]: Updated to col-md to 6 and 8. Updated col-xl to 6-6. Responsiveness looks okay.

File: buildscripts/pages/gallery.html (hunk 2 above)
Comment (ucpasas) [reply buildscripts/pages/gallery.html]: Updated to p-0 and m-1 as per specification.

File: buildscripts/pages/gallery.html (hunk 3 above)
Comment (ucpasas) [reply buildscripts/pages/gallery.html]: Reverted back to South-East Queensland as above updates fixed responsiveness issues.

**Review by OsiJr — APPROVED** [review]

**Pull request merged**